tests/Sharpliner.Tests/AzureDevOps/Model/Tasks/TaskBuilderTests.cs

```diff
@@ -101,7 +101,7 @@ private class PowershellTaskPipeline : TestPipeline
                     {
                         Powershell.FromResourceFile("Sharpliner.Tests.AzureDevOps.Resources.Test-Script.ps1"),
                         Powershell.FromResourceFile("Test-Script.ps1"),
-                        Powershell.Inline("Connect-AzContext", "Set-AzSubscription --id foo-bar-xyz"),
```

Comment:
this was misleading as in order to use the azure-sdk we need to use [AzurePowerShell@5](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/azure-powershell-v5?view=azure-pipelines)